docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -312,6 +312,11 @@ Once you have completed these steps, you are finished with the deprecation.
 In each :term:`feature release <Feature release>`, all
 ``RemovedInDjangoXXWarning``\s matching the new version are removed.
 
+The :mod:`!django.utils.deprecation` module provides some helpful deprecation
+utilities, such as a ``@deprecate_posargs`` decorator to assist with converting
+positional-or-keyword arguments to keyword-only. See the inline documentation
+in the module source.
```

Comment:
Noting that I think this is sufficient docs
Everything that is stated above is still true when using the helper (you still need to test the warning etc) so it is just a side note.